app/views/content/train-to-be-a-teacher/how-to-apply-for-teacher-training.md

```diff
@@ -124,26 +122,17 @@ Choose referees who could comment on things like your:
 
 Your referee will also be asked if they know of any reason why you should not work with children.
 
-Ideal referees could include:
-
-* your university tutor or supervisor
-* your current line manager at work
-* your previous employer
-* a teacher at a school where you work or volunteer
-* a supplier or client you’ve worked with (if you’re self-employed)
-
-Referees should not be family members, partners or friends.
+You'll need different referees depending on your circumstances. If you:
```

Comment:
```suggestion
You'll need different types of referees depending on your circumstances. If you:
```

I wonder if this maybe needs rewording a bit as suggested or something like: 

"The type of referee you need depends on your circumstances"